Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -2981,6 +3017,51 @@ bool RenderLayerBacking::paintsBoxDecorations() const
     return !supportsDirectlyCompositedBoxDecorations(renderer());
 }
 
+#if HAVE(HDR_SUPPORT)
+bool RenderLayerBacking::paintsHDRContent(RenderLayer::PaintedContentRequest& request) const
+{
+    auto& settings = renderer().settings();
+    if (!settings.hdrForImagesEnabled() && !settings.canvasPixelFormatEnabled())
+        return false;
+
+    auto hasPaintedHDRContent = [&settings = settings] (RenderElement& renderer) -> bool {
```

Comment:
Likely doesn't need `-> bool`

Reply:
This is fixed in https://github.com/WebKit/WebKit/pull/40360.